scripts/update-editors-choice-apps-inputlist

```diff
@@ -1,27 +1,27 @@
 org.mozilla.firefox
 org.gnome.Builder
 com.github.johnfactotum.Foliate
-org.libreoffice.LibreOffice
 org.kde.kdenlive
 com.slack.Slack
 org.inkscape.Inkscape
 org.blender.Blender
 org.darktable.Darktable
-org.videolan.VLC
 org.gabmus.hydrapaper
-org.filezillaproject.Filezilla
 org.kde.krita
-org.gimp.GIMP
-com.spotify.Client
```

Comment:
I think a few of these are terrible removals also. This is a very popular and well working app.

Reply:
That's precisely why they don't need to be in editor's choice all the time. People already know these apps, and they're likely to be in the "popular" list as well.

The idea is that we'd update editor's choice regularly, and feature popular apps like GIMP sometimes, but not permanently.

Reply:
That does actually make sense.